### DISCUSSION TITLE
Way to open file with external application from file picker

### DISCUSSION THREAD

**Discussion by A-Walrus** [helix-editor/helix#3532] [Ideas]
Say there is some non-text file in the project, for example an image, it would be nice to be able to open it from helix's file picker externally in whatever application is the default on the system, possibly with a crate like: [open](https://crates.io/crates/open).

**Comment by CBenoit** [comment]
You probably want to check https://github.com/lotabout/skim or https://github.com/junegunn/fzf
You can do exactly that
